test/local/verifier_upgrade_tests.js

```diff
@@ -5,7 +5,7 @@
 var test = require('../ptaptest')
 var TestServer = require('../test_server')
 var Client = require('../client')
-var createDBServer = require('fxa-auth-db-mem')
+var createDBServer = require('fxa-auth-db-mysql')
```

Comment:
This looks weird because of the dependency name, but `index.js` in `db-mysql` is actually equivalent to the old `index.js` from `db-mem`.